alpaca/broker/requests.py

```diff
@@ -891,11 +891,21 @@ class BatchJournalRequestEntry(NonEmptyRequest):
 
     Attributes:
         to_account (UUID): Account to fund in batch journal request.
-        amount (float): The cash amount in USD to fund by.
+        amount (Union[str, float]): The cash amount in USD to fund by.
+        transmitter_name (Optional[str]): For cash journals, travel rule related name info.
+        transmitter_account_number (Optional[str]): For cash journals, travel rule account number info.
+        transmitter_address (Optional[str]): For cash journals, travel rule related address info.
+        transmitter_financial_institution (Optional[str]): For cash journals, travel rule related institution info.
+        transmitter_timestamp (Optional[str]): For cash journals, travel rule related timestamp info.
     """
 
     to_account: UUID
-    amount: float
+    amount: Union[str, float]
+    transmitter_name: Optional[str] = None
+    transmitter_account_number: Optional[str] = None
+    transmitter_address: Optional[str] = None
+    transmitter_financial_institution: Optional[str] = None
+    transmitter_timestamp: Optional[str] = None
```

Comment:
Does this endpoint also accept a `description` per individual journal in the batch, similar to the [single create journal endpoint](https://docs.alpaca.markets/reference/createjournal-1)?

Reply:
Good catch! Please let me check and add it if it is acceptable.

Reply:
added!